### PR TITLE
chore: increase tracking coverage

### PR DESCRIPTION
**Pull request by craigbilner**
Tracking was a little out of date in terms of test config so updated and added tests to increase coverage.

Fixed a bug where the child id wasn't being checked correctly.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/b7ec126a0a177f51

File: packages/tracking/__tests__/tracking.web.test.js [hunk 1]
```diff
@@ -2,10 +2,14 @@ import { View, Text } from "react-native";
 import React from "react";
 import PropTypes from "prop-types";
 import renderer from "react-test-renderer";
+import Enzyme, { mount } from "enzyme";
```
Comment (colincclark) [inline packages/tracking/__tests__/tracking.web.test.js]: Might be worth using Jest `setupFiles` array to import these dependencies globally so we don't have to reproduce these imports in each test file? Probably a separate PR though...wdyt?

**Comment by colincclark** [comment]
also, you can use the conventionalcommit type `test` rather than `chore`

File: packages/tracking/__tests__/tracking.web.test.js (hunk 1 above)
Comment (colincclark) [reply packages/tracking/__tests__/tracking.web.test.js]: https://github.com/newsuk/times-components/pull/524 will finish this off if you think it is worth continuing with?

**Pull request merged**